.github/workflows/test-release.yaml

```diff
@@ -0,0 +1,55 @@
+name: Test Semantic Release
+
+on: workflow_dispatch
+
+jobs:
+  #  release:
+  #    runs-on: ubuntu-latest
+  #    steps:
+  #      - uses: actions/checkout@v2
+  #        with:
+  #          fetch-depth: 0
+  #          persist-credentials: true
+  #
+  #      - uses: actions/setup-node@v2
+  #        with:
+  #          node-version: '14'
+  #
+  #      - run: yarn install --frozen-lockfile
+  #
+  #      - name: Import GPG key
+  #        uses: crazy-max/ghaction-import-gpg@v3.0.1
+  #        with:
+  #          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+  #          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+  #          git-user-signingkey: true
+  #          git-commit-gpgsign: true
+  #
+  #      - name: Build
+  #        run: yarn run build
+  #
+  #      - run: npx semantic-release
+  #        env:
+  #          GITHUB_TOKEN: ${{ secrets.ADMIN_ACCESS_TOKEN }}
+  #          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+  #          GIT_COMMITTER_NAME: github-actions[bot]
+  #          GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMITTER_EMAIL }}
+  update-tokens:
+    name: Update public JSONBin
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: get tokens data

```

Comment:
This data isn't being used?

Reply:
Nope. I just thought that every step has to have a name.